zarr/tests/test_storage.py

```diff
@@ -1802,12 +1802,24 @@ def test_context_manager(self):
             store[self.root + 'baz'] = b'qux'
             assert 2 == len(store)
 
+    def test_delitem(self):
+        store = self.create_store()
+        store[self.root + 'foo'] = b'bar'
+
+        del store[self.root + 'foo']
+
+        assert store[self.root + 'foo'] == b''
+
     def test_pop(self):
         # override because not implemented
         store = self.create_store()
         store[self.root + 'foo'] = b'bar'
-        with pytest.raises(NotImplementedError):
-            store.pop(self.root + 'foo')
+
+        store.pop(self.root + 'foo')
+
+        assert store[self.root + 'foo'] == b""
+        # with pytest.raises(NotImplementedError):
+        #     store.pop(self.root + 'foo')
 
     def test_popitem(self):
         # override because not implemented
```

Comment:
Just realized we can actually drop these methods since the `StoreTests` base class already includes them. So would go ahead and delete them. Though we can include any relevant changes in the [`test_pop`]( https://github.com/I-am-Emmanuel/zarr-python/blob/0144f953e99ec73f4d03a7e6825ea48578e9e600/zarr/tests/test_storage.py#L173 ) and [`test_popitem`]( https://github.com/I-am-Emmanuel/zarr-python/blob/0144f953e99ec73f4d03a7e6825ea48578e9e600/zarr/tests/test_storage.py#L195 ) methods above.

Reply:
Okay. I will do that. Thanks